### ISSUE TITLE
thread 'tokio-runtime-worker' has overflowed its stack

### ISSUE THREAD

**Issue by johansmitsnl** [flathub/com.belmoussaoui.Authenticator#9]
Since the update I'm unable to start the app.

```
$ flatpak run com.belmoussaoui.Authenticator

(process:2): Gtk-WARNING **: 12:42:00.327: Locale not supported by C library.
	Using the fallback 'C' locale.
 INFO  authenticator::application > Authenticator (com.belmoussaoui.Authenticator)
 INFO  authenticator::application > Version: 4.1.0 ()
 INFO  authenticator::application > Datadir: /app/share/authenticator
 INFO  authenticator::models::database > Running DB Migrations...
 INFO  authenticator::models::database > Database pool initialized.

thread 'tokio-runtime-worker' has overflowed its stack
fatal runtime error: stack overflow
```

**Comment by bilelmoussaoui** [comment]
This was fixed in 4.1.1 already, see https://gitlab.gnome.org/World/Authenticator/-/issues/312

**Comment by johansmitsnl** [comment]
Noticed there is already a issue https://gitlab.gnome.org/World/Authenticator/-/issues/312
